packages/typespec-azure-resource-manager/lib/common-types/managed-identity-ref.tsp

```diff
@@ -0,0 +1,59 @@
+import "./managed-identity.tsp";
+
+using Azure.ResourceManager.CommonTypes.Private;
+
+namespace Azure.ResourceManager.CommonTypes;
+
+/**
+ * The properties of the managed service identities assigned to this resource.
+ */
+@@armCommonDefinition(ManagedServiceIdentity,
```

Comment:
I thought we decided not to separate the `@armCommonDefinition` and `@armCommonParameter` decorators into a separate file?

Reply:
I think we need to separate them as not to significantly change the autorest for common-types swagger generation FOR NOW, otherwise $ref will be generated.